modules/core/shared/src/main/scala/lucuma/core/math/Angle.scala

```diff
@@ -300,6 +300,20 @@ object Angle extends AngleOptics {
     def -(a: Angle): Angle =
       Angle.fromMicroarcseconds(toMicroarcseconds - a.toMicroarcseconds)
 
+    /**
+    * Scalar multiplication of this angle and and natural number `a`. Exact
+    * @group Operations
+    */
+    def *(a: Long): Angle =
+      Angle.fromMicroarcseconds(toMicroarcseconds * a)
+
+    /**
+    * Scalar multiplication of this angle and double precission number `a`. Approximate
+    * @group Operations
+    */
+    def *?(a: Double): Angle =
```

Comment:
I made it `*?` to denote that is approximate. not sure if it is a good idea, WDYT?

Reply:
We just document it when it's inexact. Pretty much anything involving `Double` is inexact by definition.

Reply:
I reverted to `*`